Rakefile

```diff
@@ -113,6 +119,25 @@ rescue
   puts 'Rubocop not loaded'
 end
 
+namespace :config do
```

Comment:
As in every PR should have run this? That seems hard to enforce but we can give it a try. I still think we ought to consider treating the migrations and answer files like Rails does where the schema is the result of applying the migrations and one is not stored in the git repo.